Modules/FindCUDA.cmake

```diff
@@ -945,7 +977,9 @@ set(CUDA_TOOLKIT_TARGET_DIR_INTERNAL "${CUDA_TOOLKIT_TARGET_DIR}" CACHE INTERNAL
 set(CUDA_SDK_ROOT_DIR_INTERNAL "${CUDA_SDK_ROOT_DIR}" CACHE INTERNAL
   "This is the value of the last time CUDA_SDK_ROOT_DIR was set successfully." FORCE)
 
-include(${CMAKE_CURRENT_LIST_DIR}/FindPackageHandleStandardArgs.cmake)
+#include(${CMAKE_CURRENT_LIST_DIR}/FindPackageHandleStandardArgs.cmake)
+include(${CMAKE_ROOT}/Modules/FindPackageHandleStandardArgs.cmake)
```

Comment:
If I only use FindCUDA.cmake module and FindCUDA directory to shadow those of CMake, I am getting this error:
CMake Error at install/share/cmake/torch/FindCUDA.cmake:969 (include):
  include could not find load file:

    /home/bfomitchev/git/torch-android/install/share/cmake/torch/FindPackageHandleStandardArgs.cmake
Call Stack (most recent call first):
  distro/extra/cutorch/CMakeLists.txt:7 (FIND_PACKAGE)
Is there a better way to handle this ?